docking_demo.py

```diff
@@ -426,7 +427,7 @@ def main(exposure):
                 pp.pprint(cmd)
                 controller.set_command(cmd)
 
-            cv2.imshow('Features Used for Visual Servoing', color_image)
+            cv2.imshow('Features Used for Visual Servoing', fit_image_to_screen(color_image))
```

Comment:
I'd prefer that this function call override the the default 0.75 ratio with 0.5 or smaller. 0.75 was too large a display for me.